queries/futures/useGetFuturesMarkets.ts

```diff
@@ -45,12 +51,15 @@ const useGetFuturesMarkets = (options?: UseQueryOptions<FuturesMarket[]>) => {
 				FuturesMarketData.globals(),
 			]);
 
-			const enabledMarkets = markets.filter((m: any) => {
+			const filteredMarkets = markets.filter((m: any) => {
 				const asset = utils.parseBytes32String(m.asset) as FuturesMarketAsset;
-				return !!MarketKeyByAsset[asset];
+				const market = enabledMarkets.find((market) => {
```

Comment:
Just to check I understand correctly, there are certain markets that will be returned from the contract but we don't want to enable, so we have a hardcoded copy of what markets to enable on each network?

Reply:
Yes, because usually Synthetix pushes new market contracts in a paused state but it takes weeks (e.g. XMR-PERP is available but still paused since ~3 weeks) for them to be live. This avoids us having paused new markets on the UI for indefinite amounts of time.

Reply:
ok makes sense and we can't read the paused state from the contracts?